rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -354,8 +354,8 @@ public void onFailure(Throwable caught) {
      */
     private void fetchStatHistory() {
         String range = "";
-        String[] sources = new String[] { "BusyNodesCount", "FreeNodesCount", "DownNodesCount", "AvailableNodesCount",
-                                          "AverageActivity" };
+        String[] sources = new String[] { "BusyNodesCount", "FreeNodesCount", "DeployingNodesCount", "DownNodesCount",
+                                          "NumberPendingTasks", "AvailableNodesCount", "AverageActivity" };
```

Comment:
Same as in the other PR,
`PendingTasksCount` instead.
All the other metrics are XXXCount as well !